app/src/main/res/layout/fragment_session_detail.xml

```diff
@@ -17,160 +19,212 @@
-                    style="@style/TextStyle.App.Caption"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toTopOf="parent"
-                    tools:text="DAY1 / "
-                    />
+                <android.support.design.widget.CollapsingToolbarLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:fitsSystemWindows="true"
+                    app:layout_scrollFlags="scroll|exitUntilCollapsed"
+                    app:titleEnabled="false"
+                    >
 
-                <TextView
-                    android:id="@+id/session_period"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginEnd="16dp"
-                    style="@style/TextStyle.App.Caption"
-                    app:endDate="@{session.endTime}"
-                    app:layout_constraintStart_toEndOf="@id/session_day_number"
-                    app:layout_constraintTop_toTopOf="@id/session_day_number"
-                    app:layout_goneMarginStart="12dp"
-                    app:layout_goneMarginTop="16dp"
-                    app:startDate="@{session.startTime}"
-                    tools:text="10:30 - 11:00 / "
-                    />
+                    <android.support.constraint.ConstraintLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:layout_marginTop="?android:attr/actionBarSize"
+                        android:alpha="@{1.0f - toolbarTextColorFactor}"
+                        android:fitsSystemWindows="true"
+                        app:layout_collapseMode="parallax"
+                        >
 
-                <TextView
-                    android:id="@+id/session_room"
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="8dp"
-                    android:maxLines="3"
-                    android:text="@{session.room.name}"
-                    style="@style/TextStyle.App.Caption"
-                    app:layout_constraintStart_toStartOf="@id/session_day_number"
-                    app:layout_constraintTop_toBottomOf="@id/session_day_number"
-                    tools:text="Room A"
-                    />
+                        <android.support.text.emoji.widget.EmojiTextView
+                            android:id="@+id/session_title"
+                            style="@style/TextStyle.App.Title"
+                            android:layout_width="0dp"
+                            android:layout_height="wrap_content"
+                            android:layout_marginStart="16dp"
+                            android:lineSpacingExtra="8sp"
```

Comment:
I think it already defined in TextStyle.App.Title. That do you think? 🤔

Reply:
oh... It had been overlooked.
lineSpacingExtra is Deleted!!

But, I think  lineSpacingMultiplier size is a little narrow

Reply:
Thanks! 
It based material design.
We will use it and if we think it is small, we will change it👍